substrait/substrait-spark/src/test/scala/io/substrait/spark/TPCDSPlan.scala

```diff
@@ -27,6 +27,7 @@ class TPCDSPlan extends TPCDSBase with SubstraitPlanTestBase {
     sparkContext.setLogLevel("WARN")
 
     conf.setConf(SQLConf.DYNAMIC_PARTITION_PRUNING_ENABLED, false)
+    conf.setConf(SQLConf.READ_SIDE_CHAR_PADDING, false)
```

Comment:
The CI did not pass... this config is created since Spark3.4

Reply:
It's strange that all the CI builds passed when it was merged. I have reverted this PR. Thanks.

Reply:
@JkSelf Only 5 checks run in the CI for this PR, any clue why others didn't run?

Reply:
@ayushi-agarwal Can you help to re-open and try again? Thanks.